app/Foundation/ViewObjects/SystemStatusViewObject.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/*
+ * This file is part of Cachet.
+ *
+ * (c) Alt Three Services Limited
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace CachetHQ\Cachet\Foundation\ViewObjects;
+
+use CachetHQ\Cachet\Models\Component;
+use CachetHQ\Cachet\Foundation\Queries\SystemStatusQuery;
+
+class SystemStatusViewObject
```

Comment:
View object isn't really right. It's not used in the context of views exclusively?